content/blog/elsewhere/12-days/12-days.11tydata.yaml

```diff
@@ -0,0 +1,7 @@
+venue: 12 Days of Web
+url: https://12daysofweb.dev/
+image:
+  src: blog/elsewhere/12-days.jpg
+  alt: ''
```

Comment:
Does this need alt text?

Reply:
My instinct was that it's just decorative since that text is duplicated. But then again - it's not part of the title, so maybe a good idea to have up top?

Reply:
```suggestion
  alt: 12 Days of Web
```